cli/src/test/resources/ide-projects/pip/repository/uv/uv/default/uv

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "uv $*"
```

Comment:
These test files are not being used by the tests. So I would either add a run test too (not sure how to simulate that though) or remove them.
For the run test you could simply add a check for the pip argument in the uv binary and echo uv pip install f.e..